etc/testing/circle/install.sh

```diff
@@ -35,7 +35,7 @@ fi
 # To get the latest minikube version:
 # curl https://api.github.com/repos/kubernetes/minikube/releases | jq -r .[].tag_name | sort -V | tail -n1
 if [ ! -f cached-deps/minikube ] ; then
-    MINIKUBE_VERSION=v1.26.0 
+    MINIKUBE_VERSION=v1.27.0
```

Comment:
We've had some issues with caching here, we should make sure 1.27 is actually getting installed.